<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1366</width>
    <height>768</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QWidget" name="background" native="true">
    <property name="geometry">
     <rect>
      <x>-10</x>
      <y>0</y>
      <width>1960</width>
      <height>1050</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">BACKGROUND-COLOR:rgb(145, 0, 0)</string>
    </property>
    <widget class="QWidget" name="widget" native="true">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>0</y>
       <width>1950</width>
       <height>1050</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-image: url(:/images/cmsbg.jpg)</string>
     </property>
     <widget class="QLabel" name="label_4">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>1366</width>
        <height>768</height>
       </rect>
      </property>
      <property name="minimumSize">
       <size>
        <width>1336</width>
        <height>768</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>16777215</width>
        <height>16777215</height>
       </size>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="pixmap">
       <pixmap>../images/cemeterybg.jpg</pixmap>
      </property>
      <property name="scaledContents">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QTextEdit" name="txttitle_3">
      <property name="enabled">
       <bool>false</bool>
      </property>
      <property name="geometry">
       <rect>
        <x>-90</x>
        <y>60</y>
        <width>571</width>
        <height>61</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: transparent;
color:rgb(135, 0, 0);
border: none</string>
      </property>
      <property name="html">
       <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'MS Shell Dlg 2'; font-size:7.8pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-size:18pt; color:#ffffff;&quot;&gt;__________________________________________&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
     </widget>
     <widget class="QTextEdit" name="Cemasys">
      <property name="enabled">
       <bool>false</bool>
      </property>
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>20</y>
        <width>221</width>
        <height>71</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">color:WHITE;
border: none;
background-color: transparent;</string>
      </property>
      <property name="html">
       <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'MS Shell Dlg 2'; font-size:7.8pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-size:36pt; font-weight:600; color:#ffffff;&quot;&gt;C&lt;/span&gt;&lt;span style=&quot; font-size:28pt; font-weight:600; color:#ffffff;&quot;&gt;EMASYS&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
     </widget>
     <widget class="QWidget" name="frame" native="true">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>120</y>
        <width>1251</width>
        <height>581</height>
       </rect>
      </property>
      <property name="toolTipDuration">
       <number>0</number>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(238, 238, 238);
border-radius:20px;</string>
      </property>
      <widget class="QPushButton" name="backbtn">
       <property name="geometry">
        <rect>
         <x>30</x>
         <y>520</y>
         <width>111</width>
         <height>41</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">#backbtn {
background-color:rgb(143, 143, 143);
color: black;
font: 75 15pt &quot;Arial&quot;;
border-radius:15px;
}

#backbtn:hover {
color:white;
padding-left: 5rem;
background-color: rgb(113, 113, 113)
}</string>
       </property>
       <property name="text">
        <string>BACK</string>
       </property>
      </widget>
      <widget class="QTextEdit" name="Cemasys_9">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="geometry">
        <rect>
         <x>140</x>
         <y>10</y>
         <width>501</width>
         <height>71</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">color:WHITE;
border: none;
background-color: transparent;</string>
       </property>
       <property name="html">
        <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'MS Shell Dlg 2'; font-size:7.8pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot; margin-top:5px; margin-bottom:5px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-size:28pt; font-weight:600; color:#7a7a7a;&quot;&gt;Book for Interment&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
       </property>
      </widget>
      <widget class="QTextEdit" name="Cemasys_2">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="geometry">
        <rect>
         <x>40</x>
         <y>80</y>
         <width>451</width>
         <height>61</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">color:WHITE;
border: none;
background-color: transparent;</string>
       </property>
       <property name="html">
        <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'MS Shell Dlg 2'; font-size:7.8pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-size:18pt; color:#7a7a7a;&quot;&gt;Customer's Information&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="cus_fname">
       <property name="geometry">
        <rect>
         <x>40</x>
         <y>120</y>
         <width>250</width>
         <height>40</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>MS Shell Dlg 2</family>
         <pointsize>14</pointsize>
         <weight>50</weight>
         <italic>false</italic>
         <bold>false</bold>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">#cus_fname {
	font: 14pt &quot;MS Shell Dlg 2&quot;;
  background-color:white;
  border-radius: 20px;
  transition: box-shadow 0.3s;
}

#cus_fname:hover {
  border: 2px solid rgb(122, 122, 122);
  box-shadow: 0 2px 4px rgba(0, 0, 0, 0.1);
  padding: 8px;
  outline: none;
}

#cus_fname:hover:focus {
  box-shadow: 0 0 8px rgb(165, 255, 240); /* Adding a glow effect */
}
</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
       <property name="placeholderText">
        <string>First Name</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="cus_lname">
       <property name="geometry">
        <rect>
         <x>300</x>
         <y>120</y>
         <width>250</width>
         <height>40</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>MS Shell Dlg 2</family>
         <pointsize>14</pointsize>
         <weight>50</weight>
         <italic>false</italic>
         <bold>false</bold>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">#cus_lname {
	font: 14pt &quot;MS Shell Dlg 2&quot;;
  background-color:white;
  border-radius: 20px;
  transition: box-shadow 0.3s;
}

#cus_lname:hover {
  border: 2px solid rgb(122, 122, 122);
  box-shadow: 0 2px 4px rgba(0, 0, 0, 0.1);
  padding: 8px;
  outline: none;
}

#cus_lname:hover:focus {
  box-shadow: 0 0 8px rgb(165, 255, 240); /* Adding a glow effect */
}
</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
       <property name="placeholderText">
        <string>Last Name</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="mobile">
       <property name="geometry">
        <rect>
         <x>560</x>
         <y>120</y>
         <width>250</width>
         <height>40</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>MS Shell Dlg 2</family>
         <pointsize>14</pointsize>
         <weight>50</weight>
         <italic>false</italic>
         <bold>false</bold>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">#mobile {
	font: 14pt &quot;MS Shell Dlg 2&quot;;
  background-color:white;
  border-radius: 20px;
  transition: box-shadow 0.3s;
}

#mobile:hover {
  border: 2px solid rgb(122, 122, 122);
  box-shadow: 0 2px 4px rgba(0, 0, 0, 0.1);
  padding: 8px;
  outline: none;
}

#mobile:hover:focus {
  box-shadow: 0 0 8px rgb(165, 255, 240); /* Adding a glow effect */
}
</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
       <property name="placeholderText">
        <string>Mobile Number</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="txtaddress">
       <property name="geometry">
        <rect>
         <x>830</x>
         <y>120</y>
         <width>351</width>
         <height>40</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>MS Shell Dlg 2</family>
         <pointsize>14</pointsize>
         <weight>50</weight>
         <italic>false</italic>
         <bold>false</bold>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">#txtaddress {
	font: 14pt &quot;MS Shell Dlg 2&quot;;
  background-color:white;
  border-radius: 20px;
  transition: box-shadow 0.3s;
}

#txtaddress:hover {
  border: 2px solid rgb(122, 122, 122);
  box-shadow: 0 2px 4px rgba(0, 0, 0, 0.1);
  padding: 8px;
  outline: none;
}

#txtaddress:hover:focus {
  box-shadow: 0 0 8px rgb(165, 255, 240); /* Adding a glow effect */
}
</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
       <property name="placeholderText">
        <string>Address</string>
       </property>
      </widget>
      <widget class="QPushButton" name="booknowbtn">
       <property name="geometry">
        <rect>
         <x>990</x>
         <y>530</y>
         <width>240</width>
         <height>40</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">#booknowbtn {
background-color:rgb(145, 0, 0);
color: white;
font: 75 15pt &quot;Arial&quot;;
border-radius:15px;
}

#booknowbtn:hover {
color:white;
padding-left: 5rem;
background-color:rgb(180, 0, 0)
}</string>
       </property>
       <property name="text">
        <string>BOOK NOW</string>
       </property>
      </widget>
      <widget class="QTextEdit" name="txttitle_4">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="geometry">
        <rect>
         <x>40</x>
         <y>40</y>
         <width>1681</width>
         <height>61</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: transparent;
color:rgb(135, 0, 0);
border: none</string>
       </property>
       <property name="html">
        <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'MS Shell Dlg 2'; font-size:7.8pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-size:18pt; color:#7a7a7a;&quot;&gt;____________________________________________________________________________________________________&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="dec_lname">
       <property name="geometry">
        <rect>
         <x>580</x>
         <y>230</y>
         <width>260</width>
         <height>40</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>MS Shell Dlg 2</family>
         <pointsize>14</pointsize>
         <weight>50</weight>
         <italic>false</italic>
         <bold>false</bold>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">#dec_lname {
	font: 14pt &quot;MS Shell Dlg 2&quot;;
  background-color:white;
  border-radius: 20px;
  transition: box-shadow 0.3s;
}

#dec_lname:hover {
  border: 2px solid rgb(122, 122, 122);
  box-shadow: 0 2px 4px rgba(0, 0, 0, 0.1);
  padding: 8px;
  outline: none;
}

#dec_lname:hover:focus {
  box-shadow: 0 0 8px rgb(165, 255, 240); /* Adding a glow effect */
}
</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
       <property name="placeholderText">
        <string>Last Name</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="dec_fname">
       <property name="geometry">
        <rect>
         <x>40</x>
         <y>230</y>
         <width>260</width>
         <height>40</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>MS Shell Dlg 2</family>
         <pointsize>14</pointsize>
         <weight>50</weight>
         <italic>false</italic>
         <bold>false</bold>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">#dec_fname {
	font: 14pt &quot;MS Shell Dlg 2&quot;;
  background-color:white;
  border-radius: 20px;
  transition: box-shadow 0.3s;
}

#dec_fname:hover {
  border: 2px solid rgb(122, 122, 122);
  box-shadow: 0 2px 4px rgba(0, 0, 0, 0.1);
  padding: 8px;
  outline: none;
}

#dec_fname:hover:focus {
  box-shadow: 0 0 8px rgb(165, 255, 240); /* Adding a glow effect */
}
</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
       <property name="placeholderText">
        <string>First Name</string>
       </property>
      </widget>
      <widget class="QTextEdit" name="Cemasys_4">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="geometry">
        <rect>
         <x>40</x>
         <y>190</y>
         <width>451</width>
         <height>61</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">color:WHITE;
border: none;
background-color: transparent;</string>
       </property>
       <property name="html">
        <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'MS Shell Dlg 2'; font-size:7.8pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-size:18pt; color:#7a7a7a;&quot;&gt;Decease's Information&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="dec_mname">
       <property name="geometry">
        <rect>
         <x>310</x>
         <y>230</y>
         <width>260</width>
         <height>40</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>MS Shell Dlg 2</family>
         <pointsize>14</pointsize>
         <weight>50</weight>
         <italic>false</italic>
         <bold>false</bold>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">#dec_mname {
	font: 14pt &quot;MS Shell Dlg 2&quot;;
  background-color:white;
  border-radius: 20px;
  transition: box-shadow 0.3s;
}

#dec_mname:hover {
  border: 2px solid rgb(122, 122, 122);
  box-shadow: 0 2px 4px rgba(0, 0, 0, 0.1);
  padding: 8px;
  outline: none;
}

#dec_mname:hover:focus {
  box-shadow: 0 0 8px rgb(165, 255, 240); /* Adding a glow effect */
}
</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
       <property name="placeholderText">
        <string>Middle Name</string>
       </property>
      </widget>
      <widget class="QLabel" name="label">
       <property name="geometry">
        <rect>
         <x>120</x>
         <y>290</y>
         <width>191</width>
         <height>21</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">
font: 11pt &quot;MS Shell Dlg 2&quot;;

color:rgb(143, 143, 143)</string>
       </property>
       <property name="text">
        <string>Date of Birth</string>
       </property>
      </widget>
      <widget class="QDateEdit" name="dec_dob">
       <property name="geometry">
        <rect>
         <x>40</x>
         <y>320</y>
         <width>260</width>
         <height>40</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">font: 16pt &quot;MS Shell Dlg 2&quot;;
background-color:white;
border-radius:20px;
color: rgb(122, 122, 122)</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
       <property name="buttonSymbols">
        <enum>QAbstractSpinBox::NoButtons</enum>
       </property>
      </widget>
      <widget class="QDateEdit" name="dec_dod">
       <property name="geometry">
        <rect>
         <x>310</x>
         <y>320</y>
         <width>260</width>
         <height>40</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">font: 16pt &quot;MS Shell Dlg 2&quot;;
background-color:white;
border-radius:20px;
color: rgb(122, 122, 122)</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
       <property name="buttonSymbols">
        <enum>QAbstractSpinBox::NoButtons</enum>
       </property>
      </widget>
      <widget class="QLabel" name="label_2">
       <property name="geometry">
        <rect>
         <x>380</x>
         <y>290</y>
         <width>191</width>
         <height>21</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">
font: 11pt &quot;MS Shell Dlg 2&quot;;

color:rgb(143, 143, 143)</string>
       </property>
       <property name="text">
        <string>Date of Death</string>
       </property>
      </widget>
      <widget class="QDateEdit" name="dec_doi">
       <property name="geometry">
        <rect>
         <x>580</x>
         <y>320</y>
         <width>260</width>
         <height>40</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">font: 16pt &quot;MS Shell Dlg 2&quot;;
background-color:white;
border-radius:20px;
color: rgb(122, 122, 122)</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
       <property name="buttonSymbols">
        <enum>QAbstractSpinBox::NoButtons</enum>
       </property>
      </widget>
      <widget class="QLabel" name="label_3">
       <property name="geometry">
        <rect>
         <x>640</x>
         <y>290</y>
         <width>241</width>
         <height>21</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">
font: 11pt &quot;MS Shell Dlg 2&quot;;

color:rgb(143, 143, 143)</string>
       </property>
       <property name="text">
        <string>Date of Interment</string>
       </property>
      </widget>
      <widget class="QLabel" name="background_3">
       <property name="geometry">
        <rect>
         <x>60</x>
         <y>10</y>
         <width>71</width>
         <height>51</height>
        </rect>
       </property>
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>0</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>16777215</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color:transparent;</string>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="pixmap">
        <pixmap>../images/book.png</pixmap>
       </property>
       <property name="scaledContents">
        <bool>true</bool>
       </property>
      </widget>
     </widget>
    </widget>
   </widget>
  </widget>
 </widget>
 <resources>
  <include location="../res.qrc"/>
 </resources>
 <connections/>
</ui>
